### PR TITLE
Rashes in children

### PR DESCRIPTION
**Pull request by bottlerocket**
New rash images added. 

image

### CODE REVIEW

File: views/symptoms/rashes.html [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
        <div class="callout aside emergency" id="callout-999">
 
-            <h2 class="heading">Call 999 or go to A&amp;E if your child has a rash and they:</h2>
+            <h3>Call 999 or go to A&amp;E if your child has a rash and they:</h3>
```
Comment (Demotive) [inline views/symptoms/rashes.html:31]: Not sure it's that big a deal in a prototype, but shouldn't this be an h2?

File: views/symptoms/rashes.html [hunk 2]
```diff
@@ -47,25 +47,48 @@ <h2 class="heading">Call 999 or go to A&amp;E if your child has a rash and they:
 
       <div class="grid-2-3">
 
-            <p>The information below shouldn’t be used to self-diagnose your child’s condition, but it should give you a better idea of what is causing the rash. </p>
+           <div class="callout gp">
+
+                <h3 class="heading-large">When to see a GP</h3>
```
Comment (Demotive) [inline views/symptoms/rashes.html:52]: Not sure it's that big a deal in a prototype, but shouldn't this be an h2?

**Comment by Demotive** [comment]
Gotta say, I'm not at all convinced by doubling up the red callout 🌵

**Comment by bottlerocket** [comment]
Neither am I…but I think we need to research it. I also thought about hiding it for large screens as a strategy (at least short term)

**Comment by Demotive** [comment]
Is it more that the position of the box needs to shift quite dynamically?
I'm not convinced we need to research the presence of doubled content, I think we can hypothesise a better solution to solve the problem. It's just we are literally repeating content here - visually, but also think about screen readers etc...

**Comment by Demotive** [comment]
Merging:
• new ticket for headers across the board
• new ticket for red callouts

**Pull request merged**